site/content/docs/02-template-syntax.md

```diff
@@ -1635,11 +1635,11 @@ If `this` is falsy, no component is rendered.
 
 ---
 
-The `<svelte:element>` element lets you render an element of a dynamically specified type. This is useful for example when rich text content from a CMS. If the tag is changed, the children will be preserved unless there's a transition attached to the element. Any properties and event listeners present will be applied to the element.
+The `<svelte:element>` element lets you render an element of a dynamically specified type. This is useful for example when rich text content from a CMS. Any properties and event listeners present will be applied to the element.
```

Comment:
https://github.com/sveltejs/svelte/pull/7483#discussion_r857120120

While we're here, might as well fix it in this same PR.
```suggestion
The `<svelte:element>` element lets you render an element of a dynamically specified type. This is useful for example when displaying rich text content from a CMS. Any properties and event listeners present will be applied to the element.
```

Or w/e else fits here

Reply:
I agree.